api/src/main/java/io/druid/data/input/impl/CSVParseSpec.java

```diff
@@ -68,6 +68,17 @@ public CSVParseSpec(
     }
   }
 
+  @Deprecated
+  public CSVParseSpec(
+      @JsonProperty("timestampSpec") TimestampSpec timestampSpec,
```

Comment:
I don't think you need the `@JsonProperty` annotations here since only the main constructor is `@JsonCreator`, this legacy constructor would only be used when directly instantiating